### ISSUE TITLE
camera calibration

### ISSUE THREAD

**Issue by KungZell** [zju3dv/EasyMocap#229]
Hello, thank you for your great open source  project.@chingswy 

However, unfortunately, during camera calibration, -- out cannot visualize the result image. In fact, only the json file is generated without the effect image. Can you give me some help? Thank you very much.

**Comment by chingswy** [comment]
Are the results in `chessboard` normal?

**Comment by KungZell** [comment]
> Are the results in `chessboard` normal?
In fact, we only get the "chessboard" file, not the visual output, it seems that "--out path" to save the code does not work, below is our file directory and the result of the code generation, we do not know what the problem is and would appreciate your help.
![easymocap_data_1](https://user-images.githubusercontent.com/87838942/203752300-4e90b76a-5e2b-4781-b468-7323b3b4cfa0.PNG)
![easymocap_data_2](https://user-images.githubusercontent.com/87838942/203752350-66a5f800-358d-418e-86c3-8b74f80c15f3.PNG)
![easymocap_data_3](https://user-images.githubusercontent.com/87838942/203752480-7d68f12a-9a55-40dc-ba1b-59975bca5011.PNG)

**Comment by peiyubin** [comment]
I encountered a problem similar to yours. I run the program on the Windows operating system, and maybe this is the problem.

I found a problem with the function on line 58 and line 96 of the "detect_chessboard. py" file.

After I replaced '/images/' with '\\images\\', I successfully obtained the visualization results.
![5198fcff7bf6bd156c4b40003064e89](https://user-images.githubusercontent.com/48856288/205500407-91b1e037-d205-4137-984e-6638497a27d0.jpg)
![bbb292ed42fd88cf1e5f280b2c3f31c](https://user-images.githubusercontent.com/48856288/205500425-4b75f0ac-6b23-4500-9a98-aa65ed3d5f1c.jpg)
